### DISCUSSION TITLE
consulta sobre optimizacion de imagenes en wordpress

### DISCUSSION THREAD

**Discussion by MatiasLobelos** [community/community#110824] [Programming Help]
### Select Topic Area

Question

### Body

Buenas Tardes! 
me estoy iniciando en wordpress de hecho es la primer pagina que hago y en esta pagina https://emplearte.online/conocenos/ elementor no me deja cargar las imagenes porque dice que "no permite la carga de imagenes no optimizadas" la realidad es que ya probe optimizarlas por varios plugins y las imagenes siguen sin poder cargarse. lo mismo me pasa con la primer imagen de la pagina principal de la web que tarda en cargar y no se como optimizarlo. En si no entiendo que hacer porque el peso de las imagenes es bajo (89kb, 124kb y 149kb respectivamente) y la resolucion es 1200X675 ya que decian que esas proporciones eran las mas adecuadas. En fin no se que hacer ya que no entiendo que mas se le puede "optimizar" si es que de entrada no son pesadas ni tienen una escala grande y todas son archivos jpeg. o webp. Si alguno me puede ayudar voy a estar mas que agradecido 
PD ; tambien es la primera vez que uso github pero lei que tiene un foro muy activo 
PD2: abajo dejo las fotos en cuestion
![grupodetrabajo](https://github.com/community/community/assets/146345038/0488cc03-4757-4ccf-bbef-0189654db969)
![cientifico-analizando-celebracion-microscopio-trabajando-laboratorio-serio-concentrado-generado-inteligencia-artificial](https://github.com/community/community/assets/146345038/e2b91e94-921f-44f4-a5d4-bf03ad19a738)
![grupodetrabajo](https://github.com/community/community/assets/146345038/c0d5c98b-fddd-44cb-a716-19947812780e)
![wallpaperflare com_wallpaper](https://github.com/community/community/assets/146345038/9a9e809e-26f4-420b-af74-e45a164bc905)

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @MatiasLobelos ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the GitHub product Discussions. 

We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be closing and  locking any posts in languages other than English for now, including this one.
